x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/action_type_filter.tsx

```diff
@@ -5,42 +5,38 @@
  * 2.0.
  */
 
-import React, { useEffect, useState, useCallback } from 'react';
+import React, { useState, useCallback } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { EuiPopover, EuiFilterButton, EuiFilterSelectItem } from '@elastic/eui';
 import { ActionType } from '../../../../types';
 
 interface ActionTypeFilterProps {
   actionTypes: ActionType[];
   onChange?: (selectedActionTypeIds: string[]) => void;
+  filters: string[];
 }
 
 export const ActionTypeFilter: React.FunctionComponent<ActionTypeFilterProps> = ({
   actionTypes,
-  onChange,
+  onChange: onFilterChange,
+  filters,
 }: ActionTypeFilterProps) => {
-  const [selectedValues, setSelectedValues] = useState<string[]>([]);
   const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false);
 
-  useEffect(() => {
```

Comment:
this was causing rerenders that also caused duplicated calls to the api. Having the bug done in 2 component meant to be calling the api 4 times